src/test/java/levy/art/RijkServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package levy.art;
+
+import com.andrewoid.ApiKey;
+import levy.art.json.ArtObject;
+import levy.art.json.RijksCollection;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RijkServiceTest {
+
+    @Test
+    void getCollectionByPage() {
+        // Given
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijkService service = new RijkServiceFactory().getService();
+
+        // When
+        RijksCollection response = service.getCollectionByPage(keyString, "1").blockingGet();
+
+        // Then
+        assertNotNull(response);
+        assertNotNull(response.artObjects);
+        assertTrue(response.artObjects.length > 0);
+        ArtObject firstArtObject = response.artObjects[0];
+    }
+
+
+    @Test
+    void getCollectionByQuery() {
+        // Given
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijkService service = new RijkServiceFactory().getService();
+
+        // When
+        RijksCollection response = service.getCollectionByQuery(keyString, "flowers", 4).blockingGet();
+
+        // Then
+        ArtObject firstArtObject = response.artObjects[0];
+        assertNotNull(firstArtObject.title);
+        assertFalse(firstArtObject.title.isEmpty());
+        assertNotNull(firstArtObject.longTitle);
+        assertFalse(firstArtObject.longTitle.isEmpty());
+        assertNotNull(response);
+        assertNotNull(response.artObjects);
+        assertNotNull(response.artObjects[0]);
```

Comment:
do these lines make sense given what you are testing above it?

Reply:
add an assert for the image data.

Reply:
i did - its failing not sure why my art is not coming through - is my base url right?